### PR TITLE
KEYCLOAK-17995 Document manual migration step for Keycloak 13 upgrade

### PR DESCRIPTION
**Pull request by mhajas**

### CODE REVIEW

File: upgrading/topics/keycloak/changes.adoc [hunk 1]
```diff
@@ -38,6 +38,22 @@ Configuration changes::
 
   * The default Wildfly configuration now uses the ability to make use of an automatically generated self-signed certificate with Elytron. Refer to link:https://docs.wildfly.org/22/WildFly_Elytron_Security.html#update-wildfly-to-use-the-default-elytron-components-for-application-authentication[a dedicated `applicationSSC` server SSL context section] for details.
 
+Manual migration step needed::
```
Comment (hmlnarik) [inline upgrading/topics/keycloak/changes.adoc]: This needs to be first entry in the chapter on Migrating to 13.0.0

Comment (hmlnarik) [inline upgrading/topics/keycloak/changes.adoc]:
```suggestion
==== Manual migration step needed
```

File: upgrading/topics/keycloak/changes.adoc [hunk 2]
```diff
@@ -38,6 +38,22 @@ Configuration changes::
...
+
+  The keycloak migration script does not work out of the box after removal of SmallRye dependencies from the Wildfly container. Therefore,
+  a manual step is required to successfully upgrade to the newer version. Before running the migration script, it is necessary to manually
+  remove following lines from the configuration file.
```
Comment (hmlnarik) [inline upgrading/topics/keycloak/changes.adoc]:
```suggestion
Manual change is required when the standalone.xml contains references to any of the SmallRye modules.
SmallRye modules were removed from the underlying {appserver_name} distribution, and the server does not start
if the configuration references them. Thus if the configuration in the `standalone.xml` refers to these modules,
the server configuration migration via `migrate-standalone.cli` fails before any changes are made to the configuration.
In such case, to perform server configuration migration, you have to manually remove all the lines that refer
to SmallRye modules. In the default configuration, you need to remove specifically the following lines:
```

**Review by hmlnarik — CHANGES_REQUESTED** [review]

File: upgrading/topics/keycloak/changes.adoc [hunk 3]
```diff
@@ -38,6 +38,22 @@ Configuration changes::
 
   * The default Wildfly configuration now uses the ability to make use of an automatically generated self-signed certificate with Elytron. Refer to link:https://docs.wildfly.org/22/WildFly_Elytron_Security.html#update-wildfly-to-use-the-default-elytron-components-for-application-authentication[a dedicated `applicationSSC` server SSL context section] for details.
 
+Manual migration step needed::
+
+  The keycloak migration script does not work out of the box after removal of SmallRye dependencies from the Wildfly container. Therefore,
```
Comment (andymunro) [inline upgrading/topics/keycloak/changes.adoc]: always change "out of the box" to "by default"

**Review by andymunro — CHANGES_REQUESTED** [review]
Looks good, @hmlnarik.  Only one change requested.

**Comment by mhajas** [comment]
Thanks @hmlnarik @andymunro I updated the text.

**Review by andymunro — APPROVED** [review]
Perfectly written and very clear, @hmlnarik

**Review by hmlnarik — APPROVED** [review]
Thank you for your contribution!

**Comment by hmlnarik** [comment]
Related PR: keycloak/keycloak#7997

**Pull request merged**